test/system/fs/message_drafts_test.rb

```diff
@@ -8,7 +8,8 @@ class Fs::MessageDraftsTest < ApplicationSystemTestCase
   test "user can upload a message draft" do
     visit message_threads_path
 
-    click_link "Nahrať nové správy"
+    click_button "Vytvoriť novú správu"
+    click_link "Vytvoriť novú správu na finančnú správu"
```

Comment:
Takyto rozklik tu bude vzdy? AJ ked nemam FS alebo ked nemam UPVS?

Reply:
Zatial to tak je nastavene, ale chcela som sa spytat, ci ideme prisposobovat

Reply:
@luciajanikova dajme si to ako pending issue, zatial nechajme